docs/specification/current.md

```diff
@@ -639,6 +640,82 @@ Field | Optionality | Type | Description
 }
 ```
 
+## Feedback
```

Comment:
I think it'd be helpful, as well as consist with the rest of the spec, to add a table specifying the fields in the request to the feedback service, along with optionality, type, and description

Reply:
Dennis, this is done. Will you re-review?